main.tex

```diff
@@ -38,6 +38,7 @@ \part{Theory}\label{prt:theory}
 \input{labs/07/theory}
 \input{lectures/07-a/optimization}
 \input{lectures/07-b/energy_based_models}
+\input{labs/09/theory.tex}
```

Comment:
Same here. Have you actually tried to compile the LaTeX, or you expect me to fix bugs?

Reply:
Oops... Fixed that. Now the overleaf can compile it correctly.